azext_iot/tests/helpers.py

```diff
@@ -127,6 +132,22 @@ def create_storage_account(
     return storage_cstring
 
 
+def tags_to_dict(tags: str) -> dict:
```

Comment:
Why not reference this and remove the duplicate code in ..../tests/deviceupdate/conftest.py ?